x-pack/test/alerting_api_integration/spaces_only/tests/alerting/builtin_alert_types/es_query/rule.ts

```diff
@@ -17,30 +17,29 @@ import {
 } from '../../../../../common/lib';
 import { createEsDocuments } from '../lib/create_test_data';
 
-const ALERT_TYPE_ID = '.es-query';
-const ACTION_TYPE_ID = '.index';
-const ES_TEST_INDEX_SOURCE = 'builtin-alert:es-query';
+const RULE_TYPE_ID = '.es-query';
+const CONNECTOR_TYPE_ID = '.index';
+const ES_TEST_INDEX_SOURCE = 'builtin-rule:es-query';
 const ES_TEST_INDEX_REFERENCE = '-na-';
 const ES_TEST_OUTPUT_INDEX_NAME = `${ES_TEST_INDEX_NAME}-output`;
 
-const ALERT_INTERVALS_TO_WRITE = 5;
-const ALERT_INTERVAL_SECONDS = 3;
-const ALERT_INTERVAL_MILLIS = ALERT_INTERVAL_SECONDS * 1000;
+const RULE_INTERVALS_TO_WRITE = 5;
+const RULE_INTERVAL_SECONDS = 3;
+const RULE_INTERVAL_MILLIS = RULE_INTERVAL_SECONDS * 1000;
 const ES_GROUPS_TO_WRITE = 3;
 
 // eslint-disable-next-line import/no-default-export
-export default function alertTests({ getService }: FtrProviderContext) {
+export default function ruleTests({ getService }: FtrProviderContext) {
   const supertest = getService('supertest');
   const retry = getService('retry');
   const indexPatterns = getService('indexPatterns');
   const es = getService('es');
   const esTestIndexTool = new ESTestIndexTool(es, retry);
   const esTestIndexToolOutput = new ESTestIndexTool(es, retry, ES_TEST_OUTPUT_INDEX_NAME);
 
-  // FLAKY: https://github.com/elastic/kibana/issues/129380
-  describe.skip('alert', async () => {
+  describe('rule', async () => {
```

Comment:
This entire suite was skipped here: https://github.com/elastic/kibana/issues/129380

I commented out the specific search source type test that was flaky so that I could add a functional test for the recovery context.